packages/react/error-boundary/src/ErrorBoundaryGroup.tsx

```diff
@@ -0,0 +1,52 @@
+import { ComponentType, createContext, ReactNode, useContext, useEffect, useRef } from 'react';
+import { useIsMounted, useKey } from './hooks';
+
+const ErrorBoundaryGroupContext = createContext({ groupResetKey: {}, resetGroup: () => {} });
+if (process.env.NODE_ENV !== 'production') {
+  ErrorBoundaryGroupContext.displayName = 'ErrorBoundaryGroupContext';
+}
+
+const ErrorBoundaryGroupReset = ({ trigger }: { trigger: ComponentType<{ resetGroup: () => void }> }) => {
+  const { resetGroup } = useErrorBoundaryGroup();
+  const Trigger = trigger;
+
+  return <Trigger resetGroup={resetGroup} />;
+};
```

Comment:
I hided groupResetKey. because default ErrorBoundary can use it defaultly now.

Reply:
Thanks.